docs/import.md

```diff
@@ -1,50 +1,89 @@
-# Import
 
-GGUF models and select Safetensors models can be imported directly into Ollama.
+# Importing a model
 
-## Import GGUF
+You can import a model into Ollama:
 
-A binary GGUF file can be imported directly into Ollama through a Modelfile.
+  * From Safetensors weights; or
+  * From a GGUF file
 
-```dockerfile
-FROM /path/to/file.gguf
-```
+## Importing from Safetensors
+
+Importing your model directly from the Safetensors weights is the easiest way to import a model. Ollama supports importing several different architectures including:
+
+  * Llama (including Llama 2, Llama 3, and Llama 3.1);
```

Comment:
nit: maybe a matter of style but I rarely see semicolons (vs nothing) at the end of list items if there are bullets

Reply:
I'm following the Plain English style guide: https://www.plainenglish.co.uk/how-to-write-in-plain-english.html 

> With a list that is part of a continuous sentence, put semicolons (;) after each point and start each with a lower-case letter.